tests/tpch/generate-data.py

```diff
@@ -71,9 +171,14 @@ def generate(
             )
         print(f"Finished exporting table {table}!")
     print("Finished exporting all data!")
+    con.close()
```

Comment:
nit: you should probably use a `with closing(con)` or a `try/finally` to close the connection

Reply:
https://github.com/coiled/benchmarks/pull/1131/commits/db5388416254f2f68125ab3598957aae99cc998a ya that's better, thanks!